src/js/components/SearchHeaderChart.js

```diff
@@ -2,17 +2,18 @@
 import {useSelector} from "react-redux"
 import React from "react"
 
+import Last from "../state/Last"
 import MainHistogramChart from "./charts/MainHistogram/Chart"
-import Tab from "../state/Tab"
 import brim from "../brim"
 
 export default function SearchHeaderChart() {
-  let searchRecord = useSelector(Tab.currentEntry)
-  let chartable = true
+  const search = useSelector(Last.getSearch)
```

Comment:
Here is the first usage of the new "get actual last search record" feature. We check to see if we should show the histogram component.